abb_irb4600_support/package.xml

```diff
@@ -37,12 +37,12 @@
 
   <build_depend>roslaunch</build_depend>
```

Comment:
<build_depend>roslaunch</build_depend> should be replaced with   <depend>roslaunch</depend>

Reply:
<build_depend>roslaunch</build_depend> should be relplaced with   <depend>roslaunch</depend>

Reply:
replaced with `<test_depend>roslaunch</test_depend>`, as suggested by @gavanderhoorn